flink-cdc-connect/flink-cdc-source-connectors/flink-connector-mysql-cdc/src/main/java/org/apache/flink/cdc/connectors/mysql/MySqlValidator.java

```diff
@@ -159,6 +161,27 @@ private void checkBinlogRowImage(JdbcConnection connection) throws SQLException
         }
     }
 
+    /** Check whether the binlog row value options is empty. */
+    private void checkBinlogRowValueOptions(JdbcConnection connection) throws SQLException {
+        String rowValueOptions =
+                connection
+                        .queryAndMap(
+                                "SHOW GLOBAL VARIABLES LIKE 'binlog_row_value_options'",
+                                rs -> rs.next() ? rs.getString(2) : "")
+                        .trim()
+                        .toUpperCase();
+        // This setting was introduced in MySQL 8.0+ with default of empty string ''
+        // For older versions, assume empty string ''
+        if (!rowValueOptions.equals(BINLOG_ROW_VALUE_OPTIONS)) {
+            throw new ValidationException(
+                    String.format(
+                            "The MySQL server is configured with binlog_row_value_options %s rather than %s, which is "
+                                    + "required for this connector to work properly. Change the MySQL configuration to use a "
+                                    + "binlog_row_image='' and restart the connector.",
+                            rowValueOptions, BINLOG_ROW_VALUE_OPTIONS));
```

Comment:
I think this error message should be changed as follows.

`The MySQL server is configured with binlog_row_value_options=%s, which is possible to cause losing some binlog events for the mysql cdc connector. Please change the MySQL configuration to use a  binlog_row_image='' and rerun the job. See more details at https://dev.mysql.com/doc/refman/8.0/en/replication-features-json.html.`

Reply:
If I do not understand wrong, we do not need to change the binlog_row_image setting. Am I right ?

Reply:
> If I do not understand wrong, we do not need to change the binlog_row_image setting. Am I right ?

@ruanhang1993 
I updated log message. 

Yes. we do not need to change both settings `binlog_row_image` and `binlog_row_value_options`.